File: app/src/main/java/maderski/bluetoothautoplaymusic/asynctasks/StartServiceTask.java
package maderski.bluetoothautoplaymusic.asynctasks;

import android.content.Context;
import android.content.Intent;
import android.os.AsyncTask;

import maderski.bluetoothautoplaymusic.services.BAPMService;

/**
 * Created by Jason on 3/2/17.
 */

public class StartServiceTask extends AsyncTask<Context, Void, Void> {
    @Override
    protected Void doInBackground(Context... contexts) {
        Intent serviceIntent = new Intent(contexts[0], BAPMService.class);
        contexts[0].startService(serviceIntent);
        return null;
    }
}

File: app/src/main/java/maderski/bluetoothautoplaymusic/asynctasks/StartServiceTask.kt
package maderski.bluetoothautoplaymusic.asynctasks

import android.content.Context
import android.content.Intent
import android.os.AsyncTask

import maderski.bluetoothautoplaymusic.services.BAPMService

/**
 * Created by Jason on 3/2/17.
 */

class StartServiceTask : AsyncTask<Context, Void, Void>() {
    override fun doInBackground(vararg contexts: Context): Void? {
        val serviceIntent = Intent(contexts[0], BAPMService::class.java)
        contexts[0].startService(serviceIntent)
        return null
    }
}
